front/src/views/Boxes/components/BoxesActionsAndTable.tsx

```diff
@@ -38,54 +54,251 @@ function BoxesActionsAndTable({
     navigate(`/bases/${baseId}/boxes/${labelIdentifier}`);
 
   // --- Actions on selected Boxes
```

Comment:
@aerinsol from here until line 260 you have the texts that are showing for the alerts.